libp2p/src/main/java/io/libp2p/discovery/mdns/impl/DNSIncoming.java

```diff
@@ -27,14 +27,14 @@
  * @author Arthur van Hoff, Werner Randelshofer, Pierre Frisch, Daniel Bobbert
  */
 public final class DNSIncoming extends DNSMessage {
-    private static Logger logger                                = LogManager.getLogger(DNSIncoming.class.getName());
+    private static Logger logger                                = LoggerFactory.getLogger(DNSIncoming.class.getName());
```

Comment:
Why so many spaces before the `=`? 😅

Reply:
That was just copied sources 'as is'. If at some point we will add Java Spotless that would be fixed :)